src/Appwrite/Utopia/Database/Validator/Queries/Attributes.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Queries;
+
+class Attributes extends Base
+{
+    public const ALLOWED_ATTRIBUTES = [
+        'key',
+        'type',
+        'size',
+        'required',
+        'array'
```

Comment:
Let's add `status` and `error` here too

Reply:
added status, there is no error property in attributes